engine/clients/qdrant/search.py

```diff
@@ -34,11 +35,23 @@ def init_client(cls, host, distance, connection_params: dict, search_params: dic
     #     return "forkserver" if "forkserver" in mp.get_all_start_methods() else "spawn"
 
     @classmethod
-    def search_one(cls, vector, meta_conditions, top) -> List[Tuple[int, float]]:
+    def search_one(cls, query: Query, top) -> List[Tuple[int, float]]:
+        # Can query only one till we introduce re-ranking in the benchmarks
+        if query.sparse_vector is not None:
+            query_vector = rest.NamedSparseVector(
+                name="sparse",
+                vector=rest.SparseVector(
+                    indices=query.sparse_vector.indices,
+                    values=query.sparse_vector.values,
+                ),
+            )
+        else:
+            query_vector = query.vector
```

Comment:
nit: would be better to reverse the condition, so `query_vector = query.vector` comes first